src/main/java/picard/sam/FilterSamReads.java

```diff
@@ -276,6 +298,9 @@ protected int doWork() {
             final SamReader samReader = SamReaderFactory.makeDefault().referenceSequence(REFERENCE_SEQUENCE).open(INPUT);
             final FilteringSamIterator filteringIterator;
 
+            // Used for exclude/include tag filter which expects a List<Object> input so casting here
+            List<Object> tagList = (List) TAG_VALUE;
```

Comment:
This could be better. The problem isn't here, it's in the definition of `TagFilter`. It was a mistake to override the `Object` parameter with `List<Object>`; what the author of that change should've done is used `List<?>`, which would work for any `List`. (Actually using `Collection<?>` would be even better since the code isn't specific to `List`)

Reply:
hmm that would be nice.  Should an issue be made in htsjdk?  I don't think I would want to hold up this PR for that change though.

Reply:
ticket made in htsjdk for this change/fix - https://github.com/samtools/htsjdk/issues/1082